### ISSUE TITLE
Ausgewähltes Fahrzeug merken

### ISSUE THREAD

**Issue by Xoffroad** [Urkman/s3xy-watch-for-tesla#8]
Das einmal ausgewählte Fahrzeug im Account sollte auch bei App Neustart beibehalten werden. Das ausgewählte Fahrzeug sollte ebenso an die Watch App synchronisiert werden.
Nur das ausgewählte Fahrzeug sollte dann aktualisiert/ abgefragt werden

**Comment by Urkman** [comment]
Eine Frage dazu... Schliesst Du die App immer über den Task Manager?

**Comment by Xoffroad** [comment]
Nein. Wenn sie im Hintergrund aktiv bleibt, verändert sich die Auswahl nicht. Aber wenn der iOS die App beendet oder zb durch ein Update beendet wird, dann ist die Auswahl weg. Betrifft Watch App und iOS App gleichermaßen. Da würde ich mir einen Sync wünschen.

**Comment by Urkman** [comment]
Ich habe da gerade noch was gefunden und gefixt... Kommt mit der nächsten Version...

**Comment by Xoffroad** [comment]
Mit Version 1.3.2 ist das Thema in der App behoben. Auf der Watch bleibt allerdings immer die Liste der verfügbaren Fahrzeuge vorgeschaltet.
